### PR TITLE
PHP 8.3 compatibility

### PR DESCRIPTION
**Pull request by marcovtwout**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Tests pass?   | ✔️
| Fixed issues  | https://github.com/yiisoft/yii/issues/4552

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -108,7 +108,8 @@
 			"phpunit/phpunit": {
 				"Fix PHP 7 compatibility": "https://yiisoft.github.io/phpunit-patches/phpunit_php7.patch",
 				"Fix PHP 8 compatibility": "https://yiisoft.github.io/phpunit-patches/phpunit_php8.patch",
-				"Fix PHP 8.1 compatibility": "https://yiisoft.github.io/phpunit-patches/phpunit_php81.patch"
+				"Fix PHP 8.1 compatibility": "https://yiisoft.github.io/phpunit-patches/phpunit_php81.patch",
+				"Fix PHP 8.3 compatibility": "tests/phpunit_php83.patch"
```
Comment (samdark) [inline composer.json:112]: Could be moved to `https://yiisoft.github.io/phpunit-patches/`.

Comment (marcovtwout) [reply composer.json:112]: Ofcourse, but only if you think this is the right solution.

**Comment by mlpassos** [comment]
Hey @marcovtwout ,

Thanks for your work on Yii and the PHP community and on this... do we have a date or estimate for releasing this 8.3 compatibility? We are about to test this.. we are already on 8.1 and migrating to 8.3 now.

o//

File: composer.json (hunk 1 above)
Comment (marcovtwout) [reply composer.json:112]: @samdark Could you comment on the above? I can then see how to finish and merge this PR.

Comment (samdark) [reply composer.json:112]: Yes. I think it is worth moving there.

**Comment by samdark** [comment]
@marcovtwout merged compatibility patch.

**Review by samdark — APPROVED** [review]

**Pull request merged**